CONTRIBUTING.md

```diff
@@ -6,25 +6,30 @@ There are multiple ways to help! We rely on users around the world to help keep
 
 Please read our [Code of Conduct](CODE_OF_CONDUCT.md) before contributing. Everyone interacting with this project (or associated projects) is expected to abide by its terms.
 
+## General note
+
+The definitions for this project are housed in a submodule. Please make sure to run `git clone --recursive git@github.com/holidays/holidays`
```

Comment:
@ptrimble When i did 'make update_defs', but submodules didn't checkout.
I checked [help](https://git-scm.com/docs/git-submodule) and executed  'git submodule update --init', submodule was checked out.
So 'git submodule update --init' is needed in makefile.